signatures/rego/disk_mount.rego

```diff
@@ -30,5 +30,13 @@ tracee_match = res {
 	devname := helpers.get_tracee_argument("dev_name")
 	startswith(devname, "/dev/")
 
+	# exclude runc
+	not runc_process with input as input
+
 	res := {"mounted device": devname}
 }
+
+runc_process {
+	startswith(input.processName, "runc:")
+	input.threadId == 1
```

Comment:
might not be enough in the case that container started with pid=host (or any other pid namespace that is not new)

Reply:
Shit, I merged it.

Reply:
We can re-open the "issue" and send another PR to deal with other cases. Sorry for that.